shotover-proxy/benches/cassandra_benches.rs

```diff
@@ -1,22 +1,75 @@
-use cassandra_cpp::{stmt, Session};
+use cassandra_cpp::{stmt, Session, Statement};
 use criterion::{criterion_group, criterion_main, Criterion};
 use test_helpers::docker_compose::DockerCompose;
 
 #[path = "../tests/helpers/mod.rs"]
 mod helpers;
 use helpers::ShotoverManager;
 
+struct Query {
+    name: &'static str,
+    statement: Statement,
+}
+
 fn cassandra(c: &mut Criterion) {
     let mut group = c.benchmark_group("cassandra");
     group.throughput(criterion::Throughput::Elements(1));
     group.noise_threshold(0.2);
 
-    let statement =
-        stmt!("INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');");
+    let queries = [
+        Query {
+            name: "insert",
+            statement: stmt!(
+                "INSERT INTO benchmark_keyspace.table_1 (id, x, name) VALUES (1, 11, 'foo');"
+            ),
+        },
+        Query {
+            name: "select",
+            statement: stmt!("SELECT id, x, name FROM benchmark_keyspace.table_1;"),
+        },
+    ];
+    for query in queries {
+        // Benches the case where the message does not meet the criteria for encryption
+        #[cfg(feature = "alpha-transforms")]
+        group.bench_with_input(
+            format!("protect_local_{}_unencrypted", query.name),
+            || {
+                let compose =
+                    DockerCompose::new("examples/cassandra-protect-local/docker-compose.yml")
+                        .wait_for_n_t("Startup complete", 1, 90);
+                let shotover_manager = ShotoverManager::from_topology_file(
+                    "examples/cassandra-protect-local/topology.yaml",
+                );
+                BenchResources::new(shotover_manager, compose)
+            },
+            |b, state| {
+                b.iter(|| {
+                    state.connection.execute(&query.statement).wait().unwrap();
+                })
+            },
+        );
 
-    {
+        // Benches the case where the message does not meet the criteria for caching
         group.bench_with_input(
-            "passthrough",
+            format!("redis_cache_{}_uncached", query.name),
```

Comment:
So this is the same as the passthrough one for now, since Redis cache isn't working?